File: bank-of-web/src/main/webapp/src/app/primary-page/profile/models/userModel.ts

	
export class UserModel {

     firstName: string;
     lastName: string;
     phone: string;
     address: string;
     cnp: string;
     email:string;
     city:string;
     country:string;
     imageURL:string;
	
    constructor( firstName: string, lastName: string, phone: string, address: string, cnp: string,email:string, city: string,country:string,imageURL:string) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.phone = phone;
        this.address = address;
        this.cnp = cnp;
                this.email = email;
                        this.city = city;
                this.country = country;
                 this.imageURL = imageURL;
    }

    static fromJSONObj(inp: Array<Object>): UserModel {
        let obj=inp["data"];
        return new UserModel(obj['firstName'], obj['lastName'], obj['phone'], obj['address'], obj['cnp'], obj['email'], obj['city'], obj['country'], obj['imageURL']);
    }
}

File: bank-of-models/src/main/java/ro/fmi/bnk/models/UserModel.java
package ro.fmi.bnk.models;

public class UserModel {

    private String firstName;
	private String lastName;
	private String phone;
	private String address;
	private String cnp;
	private String email;
	private String city;
	private String country;
	private String imageURL;
	public UserModel() {
		super();
	}
	public UserModel(String firstName, String lastName, String phone, String address, String cnp, String email,
			String city, String country, String imageURL) {
		super();
		this.firstName = firstName;
		this.lastName = lastName;
		this.phone = phone;
		this.address = address;
		this.cnp = cnp;
		this.email = email;
		this.city = city;
		this.country = country;
		this.setImageURL(imageURL);
		
	}
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	public String getLastName() {
		return lastName;
	}
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	public String getPhone() {
		return phone;
	}
	public void setPhone(String phone) {
		this.phone = phone;
	}
	public String getAddress() {
		return address;
	}
	public void setAddress(String address) {
		this.address = address;
	}
	public String getCnp() {
		return cnp;
	}
	public void setCnp(String cnp) {
		this.cnp = cnp;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getCity() {
		return city;
	}
	public void setCity(String city) {
		this.city = city;
	}
	public String getCountry() {
		return country;
	}
	public void setCountry(String country) {
		this.country = country;
	}
	public String getImageURL() {
		return imageURL;
	}
	public void setImageURL(String imageURL) {
		this.imageURL = imageURL;
	}
}
